sktime/forecasting/model_selection/_tune.py

```diff
@@ -363,9 +389,14 @@ class ForecastingGridSearchCV(BaseGridSearch):
         Time (seconds) to refit the best forecaster
     scorer_ : function
         Function used to score model
+    n_best_forecasters_: list of tuples ("rank", <forecaster>)
+        The "rank" is in relation to best_forecaster_
+    n_best_scores_: list of float
+        The scores of n_best_forecasters_ sorted from best to worst
+        score of forecasters
 
     Example
```

Comment:
Need to rename to Examples (and add the extra _ on the next line), per NumPy docs convention. This doesn't fail `pydocstyle` checks (it is looking for formatting primarily), but will generate warnings in the Sphinx doc build, which makes it tough to see what's going on there.

Reply:
Can you verify that any other occurrence of "Examples" section in this code file is also adjusted (if applicable)?